### PR TITLE
add theses from Clemson

### PR DESCRIPTION
**Pull request by tjhei**
I was productive...

### CODE REVIEW

File: publications-2018.bib [hunk 1]
```diff
@@ -723,6 +723,29 @@ @article{GKK18
...
+@PhdThesis{SergeyCharnyiPhd2018,
+  Title                    = {The {EMAC} scheme for Navier-Stokes simulations, and application to flow past bluff bodies},
+  Author                   = {Sergey Charnyi},
+  School                   = {Clemson University},
+  Year                     = {2018},
+  Url                      = {https://tigerprints.clemson.edu/all_dissertations/2243/}
+}
+
+@PhdThesis{LiangZhaoPhd2018,
+  Title                    = {A numerical method for the Navier-Stokes equations in velocity-vorticity form},
```
Comment (jppelteret) [inline publications-2018.bib]: It doesn't really bother me, but the title of this thesis has different capitalisation on the linked website:
> A Numerical Method for the Navier Stokes Equations in a Velocity-Vorticity Form

Comment (drwells) [inline publications-2018.bib]: I am not sure if it matters, but the thesis title did not have a hyphen between Navier and Stokes.

Comment (tjhei) [reply publications-2018.bib]: Sigh.

Comment (jppelteret) [reply publications-2018.bib]: I think that we need to decide how pedantic we're going to be about these things.

Comment (drwells) [reply publications-2018.bib]: I cannot imagine a circumstance where the hyphen would matter. Lets not bother with fixing it.

Comment (tjhei) [reply publications-2018.bib]: No, it makes sense. I was just annoyed that his title should have a hyphen. Thanks guys!

**Pull request merged**